lib/vsc/install/shared_setup.py

```diff
@@ -967,7 +981,10 @@ def run_tests(self):
                 __import__(DEFAULT_TEST_SUITE)
             self.reload_modules(DEFAULT_TEST_SUITE)
 
-            res = TestCommand.run_tests(self)
+            try:
+                res = TestCommand.run_tests(self)
```

Comment:
We'll need to figure out why this was needed, I don't think this is OK to do...

Which version of `setuptools` were you using here @vsoch? That may be relevant since the traceback you posted actually comes from the `run_tests` in `setuptools.commad.test.TestCommand`...